app/views/content_item_signups/confirm.html.erb

```diff
@@ -11,39 +11,29 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
 
-    <h1 class="govuk-heading-l">What you’ll get</h1>
+    <h1 class="govuk-heading-l">Sign up to get emails</h1>
 
     <p class="govuk-body">
-      You'll get alerts when the government publishes or changes anything on GOV.UK about: <%= @content_item['title'] %>.
+      Get emails when information changes on GOV.UK about: <%= @content_item['title'] %>.
     </p>
 
+    <% if @content_item['description'].present? %>
+      <p class="govuk-body"><%= @subscription.description %></p>
```

Comment:
Shoudn't this be `@content_item['description']`?

Reply:
This code works (copied over as is from "new" page), but I like yours better 👍

Reply:
under the hood: https://github.com/alphagov/email-alert-frontend/blob/master/app/presenters/content_item_subscription_presenter.rb#L9

Reply:
actually no, this is the wrong action :D, let me go back to the drawing board

Reply:
ok all good now https://github.com/alphagov/email-alert-frontend/pull/732/files#diff-e59dfc3cac518fbda136a300b663a0ccR23 @vanitabarrett